src/pages/myGeotab/guides/codeBase/usingInDotnet.tsx

```diff
@@ -0,0 +1,164 @@
+import { ReactNode } from "react";
+import { Page } from "../../../../components";
+import { PageTitleProps } from "../../../../components/PageTitle/PageTitle";
+import { HeaderSections } from "../../../../components/Header/headerSectionsEnum";
+import Accordion from "../../../../components/Accordion/Accordion";
+import { TableOfContentsItem } from "../../../../components/TableOfContents/TableOfContents";
+import InformationalBox from "../../../../components/InformationalBox/InformationalBox";
+import CodeSample from "../../../../components/CodeSamplesContainer/CodeSample";
+
+const nugetPackage: ReactNode = <div>
+    The <a href="https://www.nuget.org/packages/Geotab.Checkmate.ObjectModel/" target="_blank" rel="noopener noreferrer">nuget package</a> is an SDK library for accessing MyGeotab customer databases. 
+    It is a convenient "wrapper" around Geotab's HTTP/JSON API to allow developers focus on writing code instead of moving data over the wire. It includes tools to assist authenticating against Geotab's 
+    servers, automatically serializing/deserializing JSON, and providing definitions for Checkmate object classes.
+    <br></br>
```

Comment:
Break tags are self closing
```suggestion
    <br/>
```